centreon/www/front_src/src/Dashboards/SingleInstancePage/Dashboard/Layout/Panel/ExpandableButton.tsx

```diff
@@ -0,0 +1,33 @@
+import { IconButton } from '@centreon/ui';
+import CloseIcon from '@mui/icons-material/Close';
+import { useTranslation } from 'react-i18next';
+import { ExpandableData } from './models';
+
+interface Props {
+  expandableData?: ExpandableData;
+}
+
+const ExpandableButton = ({ expandableData }: Props) => {
```

Comment:
Why don't you just pass ExpandableData as props instead of wrapping it in an object?